seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/sink/JdbcSinkWriter.java

```diff
@@ -85,7 +88,7 @@ public Optional<XidInfo> prepareCommit()
                 connectionProvider.getConnection().commit();
             }
         } catch (SQLException e) {
-            throw new IOException(e);
+            new JdbcConnectorException(JdbcConnectorErrorCode.EXCEPTION, e);
```

Comment:
Suggest use `SQL_COMMIT_FAILED`

Reply:
Whether the TRANSACTION_OPERATION_FAILED can be used

Reply:
Yes, and then add commit failure related info